online-shop/src/app/modules/shared/components/presentational/icon-button/icon-button.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+
+@Component({
+  standalone: true,
+  selector: 'app-icon-button',
+  templateUrl: './icon-button.component.html',
+  styleUrls: ['./icon-button.component.scss'],
+})
+export class IconButtonComponent {
+  @Input() iconType: string | undefined;
+  @Output() onClick = new EventEmitter<MouseEvent>();
+  public handleMouseEvent(event: MouseEvent) {
+    this.onClick.emit(event);
+  }
```

Comment:
Add some spacings here too

Reply:
And please use a return type for the defined method. `:void` would fit in this case

Reply:
done. I will write return types to all the methods, but it takes a while for me to adapt to it.